apps/server/test/integration/helpers.ts

```diff
@@ -62,13 +63,18 @@ export const before = async (context) => {
 
 	const userCard = await context.createUser(context.username);
 
+	context.sessionToken = uuid();
+
 	// Force login, even if we don't know the password
 	context.session = await context.sdk.card.create({
 		slug: `session-${userCard.slug}-integration-tests-${uuid()}`,
 		type: 'session',
 		version: '1.0.0',
 		data: {
 			actor: userCard.id,
+			token: {
+				authentication: await bcrypt.hash(context.sessionToken, 12),
```

Comment:
Do we need the generated UUID stored in `context.sessionToken` or can we just generate it here with something like `bcrypt.hash(uuid(), 12),`?

Reply:
By the way, we have a number of tests failing in this repo at the moment (unrelated to your work). I'm currently looking into them and should have them fixed by the time you want to merge this PR.

Reply:
We use `context.sessionToken` in one of our tests:

```
test('server should accept requests containing the session token in the authorization header and the session id in the x-session-id header', async () => {
	const { response } = await getSelfWithHeaders({
		Authorization: `Bearer ${context.sessionToken}`,
		'X-Session-ID': context.session.id,
	});
	...
});
```

Since login is being "forced", unless I'm missing something we wouldn't have access to the token otherwise.